src/test/java/com/easypost/Fixture.java

```diff
@@ -343,4 +344,26 @@ public static Map<String, Object> basicOrder() {
 
         return order;
     }
+
+    /**
+     * Basic end shipper address that is reusable in all test coverage.
```

Comment:
We should be exact when referencing EasyPost objects.

```suggestion
     * Basic EndShipper address that is reusable in all test coverage.
```